words.c.tpl

```diff
@@ -0,0 +1,16 @@
+#include "word-db.h"
```

Comment:
Is this file unused?

Reply:
Yes, the same template is part of the Python generate script but I wanted to make it clear what the file will look like.